api/middleware/auth0Middleware.js

```diff
@@ -0,0 +1,43 @@
+const { expressjwt: jwt } = require('express-jwt');
+const jwksRsa = require('jwks-rsa');
+const createError = require('http-errors');
+const config = require('../../config/auth0');
+
+// writing the config for expressJwt method to be invoked with product environment variables
+const verifyJwt = jwt({
+  secret: jwksRsa.expressJwtSecret({
+    cache: true,
+    rateLimit: true,
+    jwksRequestsPerMinute: 5,
+    jwksUri: config.jwksUri,
+  }),
+  audience: config.audience,
+  issuer: config.issuer,
+  algorithms: ['RS256'],
+  requestProperty: 'auth0User',
+}).unless({ path: ['/'] });
```

Comment:
Loving that we can tweak which endpoints need to be free of auth on one line - big upgrade for ease-of-use for engineers to come down the line!